requirements/edx/base.txt

```diff
@@ -39,10 +39,6 @@ elasticsearch==0.4.5
 facebook-sdk==0.4.0
 feedparser==5.1.3
 firebase-token-generator==1.3.2
-# Master pyfs has a bug working with VPC auth. This is a fix. We should switch
-# back to master when and if this fix is merged back.
-# fs==0.4.0
```

Comment:
Not that I'm aware of. Just to give a high level refresher: 

* pyfs raises an exception if AWS access key/secret key are not present. 
* boto+AWS work fine if AWS access key/secret key are not present
* I made a patch which fixes this, patched by 0.4 (which we use) and master, in a forked github repo, and notified the author of pyfs via e-mail. 
* pyfs is in Google Code which does not support PRs, so I could not create a PR. 
* I did not create a Google Code issue on pyfs, which would be a good thing to do as well.